src/main/resources/template.properties

```diff
@@ -142,3 +142,8 @@ gcs.spanner.output.database=<spanner database id>
 gcs.spanner.output.table=<spanner table id>
 gcs.spanner.output.saveMode=<Append|Overwrite|ErrorIfExists|Ignore>
 gcs.spanner.output.primaryKey=<column[(,column)*] - primary key columns needed when creating the table>
+
+# Dataplex GCS to BQ
+dataplex.gcs.bq.target.dataset=<bq target dataset>
+dataplex.gcs.bq.save.mode="errorifexists"
```

Comment:
rename to "dataplex.gcs.bq.override=true/false" ?
Don't feel strongly about this though.